rust/syntect-plugin/Makefile

```diff
@@ -4,12 +4,27 @@ XDG_CONFIG_HOME ?= $(HOME)/.config
 XI_CONFIG_DIR ?= $(XDG_CONFIG_HOME)/xi
 XI_PLUGIN_DIR ?= $(XI_CONFIG_DIR)/plugins
 
-target/release/xi-syntect-plugin:
+lang_configs = assets/*.toml
+
+out/syntect: $(lang_configs) out/gen_manifest.toml target/release/xi-syntect-plugin
+	mkdir -p out/syntect/bin
+	cp $(lang_configs) out/syntect
+	install target/release/xi-syntect-plugin out/syntect/bin
+	cp out/gen_manifest.toml out/syntect/manifest.toml
+
+out/gen_manifest.toml: Cargo.toml Cargo.lock
+	cargo run --release --example make_manifest
+
+target/release/xi-syntect-plugin: src/*.rs Cargo.toml Cargo.lock
```

Comment:
Thanks for changing this. I noticed that the makefile would never rebuild the syntect plugin if it was already built but I never got around to fixing this.